ruby/vampires.rb

```diff
@@ -0,0 +1,75 @@
+def employee_survey
+  print "- What is your name? "
+  name = gets.chomp.downcase
+
+  print "- How old are you? "
+  age = gets.chomp.to_i
+
+  print "- What year were you born? "
+  birth_year = gets.chomp.to_i
+
+  print "- Our company cafeteria serves garlic brea. Should we order some for you? "
+  garlic = gets.chomp 
+
+  print "- Would you like to enroll in the company's health insurance? "
+  insurance = gets.chomp
+
+  print "- Name any allergies you have, one at a time. When you're done or if you have none type 'done' "
+  allergy = "" # first input wont be checked if it's gets.chomp
+
+
+  conclusion = nil 
+
+  if age == (Time.now.year - birth_year) && (garlic == 'yes' || insurance == 'yes')
+    conclusion = " #{name} is probably not a vampire "
```

Comment:
These results should be at the bottom of the file.

When using a control flow it is important to complete the evaluation before returning any results. I see that you finishing with the allergy check but it would be a better practice to get the allergy information during the rest of the interview and make your evaluation after the employee answered all of the questions.

Give this a quick refactor and let me know if you have any questions.